lib/gpu/CMakeLists.txt

```diff
@@ -4,3 +4,9 @@ add_library(kotekan_gpu gpuDeviceInterface.cpp gpuEventContainer.cpp gpuProcess.
 
 target_link_libraries(kotekan_gpu PRIVATE libexternal kotekan_libs)
 target_include_directories(kotekan_gpu PUBLIC .)
+
+target_link_libraries(
+    kotekan_gpu
+    PRIVATE ${LIBEVENT_BASE} ${LIBEVENT_CORE} ${LIBEVENT_PTHREADS} ${LIBEVENT_EXTRA} event)
```

Comment:
I don't think you need to link the libraries, it's just the includes that are missing.   However given the include didn't work below.  I don't think this is a case of cmake not cross linking this correctly from kotekan_libs, but a case of MacOS having a different path that's no being picked up by FindLIBEVENT correctly.   It would a good idea to check what `${LIBEVENT_INCLUDE_DIR}` is getting set too.  Also if you need access to a MacOS system, the one in LWLab that we used for Jenkins should still be working, or let me know if you'd like me to troubleshoot on my laptop.

Reply:
If you have a moment to have a look on your MacOS laptop, that would be great!